manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s

```

Comment:
```suggestion
  SilentWithProgress: /s
InstallerSuccessCodes:
- 2
```

Reply:
![image](https://user-images.githubusercontent.com/11600822/149137813-57850fac-d32e-4a6f-8dd6-ec1ad1fbb2e4.png)

Looks like `2` is a success:
![image](https://user-images.githubusercontent.com/11600822/149137852-9d095902-d7d0-4216-ae5c-c55b47d8e09f.png)

Reply:
Thanks for this. I wasn't clear on how to resolve the issue with the exit code.